demos/python_demos/image_retrieval_demo/image_retrieval_demo/image_retrieval.py

```diff
@@ -35,16 +35,15 @@ def __init__(self, model_path, device, cpu_extension):
 
         path = '.'.join(model_path.split('.')[:-1])
         self.net = IENetwork(model=path + '.xml', weights=path + '.bin')
+        self.output_name = list(self.net.outputs.keys())[0]
         self.exec_net = ie.load_network(network=self.net, device_name=device)
 
     def predict(self, image):
         ''' Takes input image and returns L2-normalized embedding vector. '''
 
         assert len(image.shape) == 4
         image = np.transpose(image, (0, 3, 1, 2))
-        out = self.exec_net.infer(inputs={'Placeholder': image})[
-            'model/tf_op_layer_mul/mul/Normalize']
-        out = out / np.linalg.norm(out, axis=-1)
```

Comment:
Was this normalization always unnecessary? Or did MO convert the network incorrectly in previous releases?

Reply:
Just checked old model and R3. Looks like it always was unnecessary